app/event_handlers/import_uploaded_files_handler.rb

```diff
@@ -1,6 +1,6 @@
 class ImportUploadedFilesHandler
   def handle(root_object)
-    root_object.uploaded_files.each do |uploaded_file|
+    root_object.uploaded_files.includes(:file_attachment).each do |uploaded_file|
```

Comment:
Performance improvement - just a few less db calls to speed things up in the test suite